FAQ: add details on copying locked files out of dataset (#570)
datalad-handbook/book: docs/basics/101-180-FAQ.rst

When using the terminal command ``cp`` [#f1]_, it is sufficient to use the
``-L``/``--dereference`` option. This will follow symbolic links, and make
sure that content gets moved instead of symlinks.
Remember that if you are copying some annexed content out of a dataset without
unlocking it first, you will only have "read" :term:`permissions` on the files you have just 
copied. Therefore you can :
- either unlock the files before copying them out, 
- or copy them and then use the command ``chmod`` to be able to edit the file.

.. code-block:: bash

   # this will give you 'write' permission on the file
   $ chmod +w filename
   
If you are not familiar with how the ``chmod`` works (or if you forgot - let's be honest we

> all google it sometimes), this is `a nice tutorial < https://bids.github.io/2015-06-04-berkeley/shell/07-perm.html>`_ . 

Reviewer: ```suggestion
all google it sometimes), this is `a nice tutorial <https://bids.github.io/2015-06-04-berkeley/shell/07-perm.html>`_ . 
```

When using the terminal command ``cp`` [#f1]_, it is sufficient to use the
``-L``/``--dereference`` option. This will follow symbolic links, and make
sure that content gets moved instead of symlinks.
Remember that if you are copying some annexed content out of a dataset without

> unlocking it first, you will only have "read" access on the files you have just 

Reviewer: There is a glossary entry on ``permissions`` in the book. How about this:
```suggestion
unlocking it first, you will only have "read" :term:`permissions` on the files you have just 
```

The glossary entry differentiates between read, write, and execute permissions, and maybe helps to understand the greater scheme of things

<details><summary>Here is what it says </summary>
permissions

 Access rights assigned by most file systems that determine whether a user can view (read permission), change (write permission), or execute (execute permission) a specific content.

-  read permissions grant the ability to a file, or the contents (file names) in a directory.

- write permissions grant the ability to modify a file. When content is stored in the object-tree by git-annex, your previously granted write permission for this content is revoked to prevent accidental modifications.

- execute permissions grant the ability to execute a file. Any script that should be an executable needs to get such permission.

</details>

copied. Therefore you can :

> - either unlock the files before copying them out, 

Reviewer: Note to self: write a glossary entry about "unlocking"

Author: Want me to add a stub with a TODO in the glossary as part of this PR?

- or copy them and then use the command ``chmod`` to be able to edit the file.

> ```
> # this will give you 'write' permission on the file
> chmod +w filename
> ```

Reviewer: This how to get the code markup in RST:

```suggestion
.. code-block:: bash

   # this will give you 'write' permission on the file
   $ chmod +w filename
```

I would add a $ at the start of the command line for consistency across the book :)

Author: Awww. Look at past me dumping markdown in a reStructured text document. 

But now... I know ~kung-fu~ RST. Well... I have been teaching myself for the past 36 hours.

![MarvelousBitterIrishwaterspaniel-size_restricted](https://user-images.githubusercontent.com/6961185/97332566-e14e1680-187a-11eb-8bcb-044c863facd5.gif)

.. todo::

>    add a link to very simple tuto on chmod

Reviewer: Maybe: https://bids.github.io/2015-06-04-berkeley/shell/07-perm.html?

Author: Sorry for leaving this one so long on the back burner: I seemed to have missed those ntofications.

Will add the link now.
